bugwarrior/services/gmail.py

```diff
@@ -178,12 +179,28 @@ def get_labels(self):
 
     def get_threads(self):
         thread_service = self.gmail_api.users().threads()
+        threads = []
 
-        result = thread_service.list(
-            userId=self.config.login_name, q=self.config.query).execute()
-        return [
-            thread_service.get(userId='me', id=thread['id']).execute()
-            for thread in result.get('threads', [])]
+        pageToken = None
+
+        while len(threads) <= self.config.email_limit:
```

Comment:
Shouldn't this be `<` rather than `<=`? If it's equal we've already hit the limit.

Reply:
You are right, I used the pagination for a long time. Before pushing the update, it passed the quick glance.